quests.json

```
@@ -4055,7 +4301,44 @@
                 "rep": 0.03
             }
         ],
-        "objectives": [],
+        "objectives": [
+            {
+                "type": "build",
+                "target": "5a7828548dc32e5a9c28b516",
+                "number": 1,
+                "with": [
+                    {
+                        "type": "stat",
+                        "name": "durability",
+                        "value": ">=60"
+                    },
+                    {
+                        "type": "stat",
+                        "name": "ergonomics",
+                        "value": ">=58"
+                    },
+                    {
+                        "type": "stat",
+                        "name": "recoil",
+                        "value": "<=550"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "Tactical device"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "Foregrip"
+                    },
+                    {
+                        "type": "attachment",
+                        "name": "Magpul MOE M870 forestock",
+                        "id": ""
```

```suggestion
                        "type": "part",
                        "id": "5a788068c5856700137e4c8f"
```

I think having a type with a specific `part` indication versus `attachment` which is a general category of attachment would help make the information easier to parse. And the name would be inferred from the `items.{locale}.json` file